src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random randomNumber = new Random();
+    private static final ColorSupplier randomColorSupplier = new ColorSupplier();
+    private static final int maxNumber = 100;
```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;
```

![](https://cdn.ttgtmedia.com/rms/onlineimages/naming_conventions_in_java-f.png)